### PR TITLE
Package cry.1.0.0

### PR DESCRIPTION
**Pull request by toots**
### `cry.1.0.0`
OCaml client for the various icecast & shoutcast source protocols
The cry library is an implementation of the various icecast & shoutcast protocols to connect to streaming servers such as icecast

---
* Homepage: https://github.com/savonet/ocaml-cry
* Source repo: git+https://github.com/savonet/ocaml-cry.git
* Bug tracker: https://github.com/savonet/ocaml-cry/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by mseri** [comment]
Thanks!

File: packages/cry/cry.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml client for the various icecast & shoutcast source protocols"
+description:
+  "The cry library is an implementation of the various icecast & shoutcast protocols to connect to streaming servers such as icecast"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Romain Beauxis <toots@rastageeks.org>"]
+license: "GPL-2.0"
```
Comment (mseri) [inline packages/cry/cry.1.0.0/opam:8]:
```suggestion
license: "GPL-2.0-only"
```
unless I am mistaken

Comment (toots) [reply packages/cry/cry.1.0.0/opam:8]: Correct! Fixed upstream, thanks!

**Comment by mseri** [comment]
Thanks

**Pull request merged**